Resources/Prototypes/_Sunrise/Entities/Clothing/Hands/gloves.yml

```
@@ -159,3 +158,28 @@
     fiberMaterial: fibers-insulative
     fiberColor: fibers-blue
   - type: FingerprintMask
+
+- type: entity
+  parent: [ ClothingHandsGlovesColorBlack, BaseMajorContraband ]
```

YAMLlint (1.37.1)

[error] too many spaces inside brackets

(brackets)